src/Build/Definition/Project.cs

```diff
@@ -2923,7 +2923,7 @@ public override bool SetGlobalProperty(string name, string escapedValue)
                     string originalValue = (existing == null) ? String.Empty : ((IProperty)existing).EvaluatedValueEscaped;
 
                     _data.GlobalPropertiesDictionary.Set(ProjectPropertyInstance.Create(name, escapedValue));
-                    _data.Properties.Set(ProjectProperty.Create(Owner, name, escapedValue, true /* is global */, false /* may not be reserved name */));
+                    _data.Properties.Set(ProjectProperty.Create(Owner, name, escapedValue, true /* is global */, false /* may not be reserved name */, null));
```

Comment:
We should try to include the argument name when passing literals, otherwise it's not clear which parameter this maps to.

See how this line already attempts this for other arguments (badly). We could remove the /* comments */ and insert argument names for other boolean literals on this line.